README.md

```diff
@@ -438,19 +438,17 @@ owner and group are both set to httpd.
     - linux-system-roles.certificate
 ```
 
-Please note that it's also possible to use UID and GID instead of
-user and group names.
+Note that you can also use UID and GID instead of user and group names.
 
 
 ## Compatibility
 
-Currently supports CentOS 7+, RHEL 7+, Fedora.  Has been tested with Debian 10.
+Currently supports CentOS 7+, RHEL 7+, Fedora. It has been tested with Debian 10.
 
 
 ## Dependencies
 
-The python packages `pyasn1`, `cryptography` and `dbus-python` are required but automatically installed by the role.
-In addition to those the `certmonger` provider (default) also depends on certmonger package and service (also installed and configured by the role).
+No dependencies so far.
```

Comment:
Just say `None`.

Reply:
otherwise, lgtm

Reply:
Updated. I'll leave as a draft for now because I might do more updates tomorrow based on Eliane's feedbacks.